src/slic3r/GUI/Tab.cpp

```diff
@@ -3622,9 +3627,17 @@ void TabFilament::toggle_options()
         toggle_line("adaptive_pressure_advance_model", has_adaptive_pa && pa);
         toggle_line("adaptive_pressure_advance_bridges", has_adaptive_pa && pa);
 
+<<<<<<< HEAD
```

Comment:
There's a left-over merge conflict here

Reply:
Oops, forgot to force-push after I fixed that, should be good now